Fix #23 -- implement output message template format (#29)
codingjoe/relint: README.md

git diff --unified=0 | relint my_file.py --diff
```

> ### custom message format

Reviewer: ```suggestion
### Custom message format
```

> You can customize the output message format by using the `--msg-template=<format string>` option.

Reviewer: ```suggestion
Customize the output message format with the `--msg-template=<format string>` option.
```

> The format string uses the [Python format syntax](https://docs.python.org/3/library/string.html#formatstrings) and the following fields are available:

Reviewer: ```suggestion
[Python format syntax](https://docs.python.org/3/library/string.html#formatstrings)
is suported for the message template and the following fields are available:
```
